### PR TITLE
Testing maths magician with jest.js api

### PR DESCRIPTION
**Pull request by learnwithalfred**
### Add test to operate and calculate functions
Screenshot 2022-07-05 at 22 39 20

### CODE REVIEW

**Comment by learnwithalfred** [comment]
Good job @bruk19

File: src/logic/operate.test.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  const operation = '+';
+  expect(operate(num1, num2, operation)).toBe('4');
+});
+
+test('Multiply two numbers', () => {
+  const num1 = 2;
+  const num2 = 2;
+  const operation = 'x';
+  expect(operate(num1, num2, operation)).toBe('4');
+});
```
Comment (julie-ify) [inline src/logic/operate.test.js]: - [ ] Good job so far, however, kindly ensure that your tests also cover the following;

 - There should be the usage of Jest snapshots for testing all the components. Here is a[ link](https://jestjs.io/docs/snapshot-testing) to learn how to do this.
 - There should be the usage of React Testing Library to simulate user interaction. This[ link](https://github.com/testing-library/react-testing-library) will help you learn how to write this test.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @learnwithalfred and @bruk19 

Good job so far!

## Highlights!
- Good feature branch name ✔️ 
- No linter errors ✔️ 
- Descriptive pull request ✔️ 
- Descriptive commit messages ✔️ 
- Descriptive README ✔️ 
- Tests passed ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by t-yanick — APPROVED** [review]
Hi @learnwithalfred, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**